.changeset/chilled-toys-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@journeyapps/react-native-quick-sqlite': major
```

Comment:
It could be a nice time to change the package name at the same time

Reply:
Package name change requires additional work such (I think last time we tried we had peerDependency issues, or with the app easily having two copies of the lib after updating). So will do that separately.